backend/migrations/versions/4b77ac3913c1_add_last_leaderboard_reward_date_to_.py

```
@@ -0,0 +1,38 @@
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4b77ac3913c1'
+down_revision = '647cfc4c0735'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Ruff (0.8.2)

Missing return type annotation for public function `upgrade`

Add return type annotation: `None`

(ANN201)